lib/govuk_index/publishing_event_processor.rb

```diff
@@ -0,0 +1,10 @@
+require 'govuk_index/publishing_event_worker'
+
+module GovukIndex
+  class PublishingEventProcessor
+    def process(message)
+      PublishingEventWorker.perform_async(message.payload)
+      message.ack
```

Comment:
Do we have to worry about the case where we fail to add the message to the Sidekiq queue?